commands/fun/trivia.js

```diff
@@ -0,0 +1,67 @@
+const Social = require(`${process.cwd()}/base/Social.js`);
+const Discord = require("discord.js");
+const snek = require("snekfetch");
+const B64 = require("base-64");
```

Comment:
You don't need this dependency if you don't request the data from the API in base64.